test/kwokctl/kwokctl_kind_logs.test.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+DIR="$(realpath "${DIR}")"
+
+source "${DIR}/helper.sh"
+
+function main() {
+  local all_releases=("${@}")
+
+  test_all "kind" "logs" "${all_releases[@]}" || exit 1
```

Comment:
This test will fail because `/tmp/kwok.log` is not going to be present
One solution is to read config the same way it is being done in `components.BuildKwokControllerComponent`
That way, we can extract log mounts and add them to extra mounts.

I just wanted to confirm that this is okay, or is there another way I should approach this? :relaxed: